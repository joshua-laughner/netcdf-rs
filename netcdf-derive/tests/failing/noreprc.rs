#[derive(netcdf_derive::NcType)]
struct NoReprC {
    a: u8,
}

fn main() {}
