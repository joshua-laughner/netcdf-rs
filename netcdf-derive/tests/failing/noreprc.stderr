error: Can not derive NcType for struct without fixed layout
 --> tests/failing/noreprc.rs:2:1
  |
2 | / struct NoReprC {
3 | |     a: u8,
4 | | }
  | |_^
  |
  = help: struct must have attribute #[repr(C)]
